ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/featuredcategorylist/v1/featuredcategorylist/clientlibs/dist/css/categorylist.css

```diff
@@ -0,0 +1,51 @@
+.cmp-categorylist {
+    padding: 1rem;
+}
+
+.cmp-categorylist .cmp-categorylist-content {
```

Comment:
The entire css file does not use the correct BEM style I think.

Reply:
Fixed now.